sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/reflection/KalixMethod.scala

```diff
@@ -89,35 +90,32 @@ case class VirtualServiceMethod(component: Class[_], methodName: String, inputTy
 }
 
 case class CombinedSubscriptionServiceMethod(
+    componentName: String,
     combinedMethodName: String,
-    representative: SubscriptionServiceMethod,
     methodsMap: Map[String, Method])
     extends AnyJsonRequestServiceMethod {
 
   val methodName = combinedMethodName
-
-  val javaMethod = representative.javaMethod
-
-  val inputType: Class[_] = javaMethod.getParameterTypes()(representative.inputTypeParamIndex)
+  override def inputType: Class[_] = classOf[ScalaPbAny]
 
   override def requestMethod: RequestMethod = RequestMethod.POST
-  override def javaMethodOpt: Option[Method] = Some(javaMethod)
+  override def javaMethodOpt: Option[Method] = None
 
-  val pathTemplate = buildPathTemplate(javaMethod.getDeclaringClass.getName, methodName)
+  val pathTemplate = buildPathTemplate(componentName, methodName)
+
+  val streamIn: Boolean = false
+  val streamOut: Boolean = false
```

Comment:
I think there were some suboptimal stuff here. 

A `CombinedSubscriptionServiceMethod` should...

* always have the `inputType` set to proto Any, that's what it will be at the end
* it can't have a javaMethod because is a synthetic one, we were using the one from the `representative`, but that's the wrong one.
* it can't be streamIn nor streamOut since it's a subscription one

Anyway, by removing the option to pass the view state in the update method, there are a bunch of code here that becomes obsolete and gives us the opportunity to clean it up.